src/main/java/com/code_intelligence/jazzer/driver/OfflineInstrumentor.java

```diff
@@ -55,11 +55,14 @@ public static boolean instrumentJars(List<String> jarLists) {
 
     List<String> errorMessages = new ArrayList<>();
     for (String jarPath : jarLists) {
+      String jarBaseName =
```

Comment:
Come to think of it, `cp` could [contain directories](https://docs.oracle.com/javase/8/docs/technotes/tools/windows/classpath.html#A1100762), in which case all jars in those are included in the classpath. As that's probably not the intended use-case here, we could ignore directories altogether. That could lead to some confusion those. 

Another option could be to change the flag `instrument_only` to contain the list of jar files and not "misuse" `cp`. WDYT?

Reply:
Good idea, I updated this to use --instrument_only=<list_of_jars>